bazel/envoy_test.bzl

```diff
@@ -108,13 +98,12 @@ def envoy_cc_fuzz_test(
         tags = tags,
         **kwargs
     )
+
+    raw_binary_name = name + "_raw"
     cc_test(
-        name = name,
-        copts = fuzz_copts + envoy_copts("@envoy", test = True),
-        linkopts = _envoy_test_linkopts() + select({
-            "@envoy//bazel:libfuzzer": ["-fsanitize=fuzzer"],
-            "//conditions:default": [],
-        }),
+        name = raw_binary_name,
```

Comment:
why does the raw binary name get compiled as a cc_test?

Reply:
Ah, good question - so I left the original binary as a `cc_test` since I didn't get the chance to look at coverage support, so I left in place the original cc_test + its data dependency on the corpus so we can still use the old way of collecting code coverage. Maybe we can migrate this part to `bazel_rules` in a subsequent step, or you think we should do it now?

Reply:
Hmmm fuzz coverage actually uses the libfuzzer linked target (and normal test coverage ignores fuzzers).

Let me clone locally and see what happens when running fuzz_coverage. It might work since the fuzz coverage script expects a libfuzzer linked binary but the tag might need to change to filter for `fuzz_target_binary` targets.